web-stories.php

```diff
@@ -60,15 +65,155 @@
 	define( 'WEBSTORIES_DEV_MODE', false );
 }
 
+/**
+ * Setup web stories compatibility class.
+ *
+ * @SuppressWarnings(PHPMD.MissingImport)
+ *
+ * @since 1.2.0
+ *
+ * @return Web_Stories_Compatibility
+ */
+function get_web_stories_compat_instance() {
```

Comment:
Nit: Rename to `web_stories_get_compat_instance`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5212/commits/41a4bc1a6af21dac139840117c411d0abb6b2c3c